extensions/Worker.Extensions.Abstractions/src/FixedDelayRetryAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public class FixedDelayRetryAttribute : RetryAttribute
+    {
+        public const string Strategy = "fixedDelay";
+
+        public FixedDelayRetryAttribute(int maxRetryCount, string delayInterval)
+        {
+            MaxRetryCount = maxRetryCount;
+            DelayInterval = delayInterval;
```

Comment:
Should we validate the delayInterval param value is in the specific format?

Reply:
Let me investigate to see if we should handle it here or elsewhere - going off of [the Timer Trigger attribute](https://github.com/Azure/azure-functions-dotnet-worker/blob/bb7aae9c07baca52004877db452762344f0db895/extensions/Worker.Extensions.Timer/src/TimerTriggerAttribute.cs#L31) where the schedule isn't validated on construction. Not sure yet what it looks like e2e if we validate here and throw something like a FormatException.

cc: @fabiocav @brettsam for thoughts

Reply:
The inproc(webjobs) version uses `TimeSpan.TryParse` for basic validation. We could consider adopting that.

Reply:
Yes will be adopting it - @fabiocav and I looked at what validations should exists for this and when they should throw - the inproc has this validation and [this one](https://github.com/Azure/azure-webjobs-sdk/blob/3fbe3f35fcfd451865700de264b72694b6df2253/src/Microsoft.Azure.WebJobs.Host/Timers/FixedDelayStrategy.cs#L15). For other OOP workers functions.json files also get validated.